### DISCUSSION TITLE
Cant use value from body in my new Api

### DISCUSSION THREAD

**Discussion by elVengador** [vercel/next.js#37497] [Help]
```js
export default function handler(req: NextApiRequest, res: NextApiResponse) {
    const { name } = req.body
    console.log('name', name, req.body)
    res.status(200).json({ name: 'John Doe' })
}
```
and when I print `req.body` has the correct value, but when I try to access is undefined, what is wrong?
![image](https://user-images.githubusercontent.com/27080940/172288670-6ce936e6-eb92-41a8-894f-0d394f309888.png)

**Comment by icyJoseph** [comment]
It is coming as text, either:

- Include `content-type: 'application/json'` when you send the request, or
- Use `JSON.parse` on the body, sort of like this: `const {name} = JSON.parse(req.body)`
